keel-ec2/src/main/kotlin/com/netflix/spinnaker/keel/ec2/asset/EC2TypeConverter.kt

```diff
@@ -33,7 +33,11 @@ class EC2TypeConverter(
         builder.region = securityGroup.region
         builder.vpcName = securityGroup.vpcId?.let { cloudDriverCache.networkBy(it).name }
         builder.description = securityGroup.description
-        securityGroup.inboundRules.forEach { rule ->
+        securityGroup.inboundRules
+          .flatMap { rule ->
+            rule.portRanges?.map { rule to it } ?: listOf(rule to null)
+          }
```

Comment:
Here's where I map across the multiple port ranges River may have in its model of the rule and turn them into multiple distinct rules